code/obj/item/clothing/hats.dm

```diff
@@ -846,6 +846,18 @@
 	icon_state = "tophat"
 	w_class = 4
 
+/obj/item/clothing/head/bighat/hoc_chefhat
+	name = "Head Chef's hat"
+	desc = "The ultimate symbol of culinary mastery, setting you a step above the rest of Catering."
+	icon_state = "hoc-chef"
+	item_state = "hoc-chef"
+	c_flags = SPACEWEAR
```

Comment:
Should this really be spacewear?

Reply:
When I looked through the existing hats, for some reason the regular chefs hats are spacewear and I didn't want to touch it in case there was some weird reason for it

Reply:
Probably oldcode dependency on the variable before flags were implemented to hide hair. Best to remove.

Consider adding `seal_hair = 1` if you desire that functionality.